static/code/editor.js

```diff
@@ -953,7 +953,90 @@ Code.functionEditor.focus = function(userAction) {
 };
 
 ////////////////////////////////////////////////////////////////////////////////
-//Code.jsspEditor = new Code.GenericEditor('JSSP');
+Code.jsspEditor = new Code.GenericEditor('JSSP');
+
+/**
+ * JavaScript Server Page editor.  Does not exist until tab is selected.
+ * @type {Object}
+ * @private
+ */
+Code.jsspEditor.editor_ = null;
+
+/**
+ * Create the DOM for this editor.
+ * @param {!Element} container DOM should be appended to this containing div.
+ */
+Code.jsspEditor.createDom = function(container) {
+  container.innerHTML = `
+<style>
+#jsspEditor {
+  position: absolute;
+  top: 60px;
+  bottom: 45px;
```

Comment:
Might it not be cleaner to put editor styling in a .ass file, rather than having it inline in the .js files?

I guess if there was just one .css file for all the editors we'd load some CSS that wasn't always needed, while separating it out into a .css fie per editor and loading it dynamically would involve doing more fetches, but those do not seem like big penalties.